clients/src/main/java/org/apache/kafka/common/protocol/Protocol.java

```diff
@@ -1199,6 +1199,168 @@
 
     public static final Schema[] INIT_PRODUCER_ID_RESPONSE = new Schema[] {INIT_PRODUCER_ID_RESPONSE_V0};
 
+    public static final Schema ADD_PARTITIONS_TO_TXN_REQUEST_V0 = new Schema(
+            new Field("transactional_id",
+                    STRING,
+                    "The transactional id corresponding to the transaction."),
+            new Field("pid",
+                    INT64,
+                    "Current PID in use by the transactional id"),
+            new Field("epoch",
+                    INT16,
+                    "Current epoch associated with the PID"),
+            new Field("topic_partitions",
+                    new ArrayOf(new Schema(
+                            new Field("topic", STRING),
+                            new Field("partitions", new ArrayOf(INT32)))),
+                    "The partitions to add to the transaction")
+    );
+    public static final Schema ADD_PARTITIONS_TO_TXN_RESPONSE_V0 = new Schema(
+            new Field("error_code",
+                    INT16,
+                    "An integer error code.")
```

Comment:
No coordinator info for this responds like for `ADD_OFFSETS_TO_TXN_RESPONSE_V0` (the code aligns with the KIP -- just wondering)